docs/rules/mustache-curly-spacing.md

```diff
@@ -0,0 +1,67 @@
+# Enforce spacing on the style of mustache curly brackets. (mustache-curly-spacing)
```

Comment:
@michalsnik I was not sure about the name of this rule, there was many suggestions like: curly-bracket, mustache-curly-spacing, embedded-expression-curly-spacing but i think yours matches at least documentation.